tools/kmod/sof_remove.sh

```diff
@@ -168,6 +168,7 @@ remove_module snd_sof_dma_trace
 # when we stop testing old branches.
 remove_module snd_sof_nocodec || true
 
+remove_module snd_soc_intel_sof_realtek_common
```

Comment:
good change but in the wrong place, this should be moved to line 155

Reply:
Thanks for reminding! Already fixed it